frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -285,6 +285,9 @@ export default ({
             const { meta, data } = msgValue
             this.messages.push({
               ...createMessage({ meta, data, hash: message.hash() }),
+              // TODO: message.hash could be different from the message on event listener
+              // https://github.com/okTurtles/group-income/issues/1487
+              // so that the message.hash() can't be used as the identifier of this message
```

Comment:
The issue mentioned in this comment is closed... so should a new issue be created if this isn't going to be fixed in this PR?

Reply:
https://github.com/okTurtles/group-income/issues/1503
After we do this issue, I think we should update this. I will update comment.